selfdrive/car/torque_data/override.yaml

```diff
@@ -49,6 +49,7 @@ KIA NIRO HYBRID 2ND GEN: [2.42, 2.5, 0.12]
 KIA NIRO EV 2ND GEN: [2.05, 2.5, 0.14]
 GENESIS GV80 2023: [2.5, 2.5, 0.1]
 KIA CARNIVAL 4TH GEN: [1.75, 1.75, 0.15]
+GMC ACADIA DENALI 2018: [1.6, 1.6, 0.2]
```

Comment:
why is this in the override? I thought things started here, then strictly move to params.yaml?

Reply:
This was not a torque control car (was pid) before. Also, these parameters for the non linear cars don't mean the same. It does not matter where we put it, but just wanted to note that the "factor" does not mean anything.